GoodreadsEDA/scp_bks.py

```diff
@@ -1,86 +1,60 @@
 
 
 def get_book_urls():
     """Loads the export file from 'mk_bklst.py', and gets the urls of all
-    of the books to pull reviews for."""
+    of the books to pull reviews for. Returns the non-header rows"""
 
     with open(f"{os.getcwd()}/csv_files/booksnlinks.csv", "r") as in_file:
-        books = csv.reader(in_file, delimiter=",")
-
-        book_urls = [b for b in books]
-        return book_urls
-
-
-def scores_to_numbers(review_text):
-    """Goodreads ratings are stored as text, this function converts them to
-    their numerical values. If there is no score given, this function returns
-    None"""
+        book_urls = list(csv.reader(in_file, delimiter=","))
 
-    if review_text == "it was amazing":
-        review_score = 5
-    elif review_text == "really liked it":
-        review_score = 4
-    elif review_text == "liked it":
-        review_score = 3
-    elif review_text == "it was ok":
-        review_score = 2
-    elif review_text == "did not like it":
-        review_score = 1
-    else:
-        return None
-
-    return review_score
+    return book_urls[1:]
 
 
 def replace_double_quotes(string):
     """When entering information into the SQL db, there is a chance to get
     double quote characters within the text. This turns them into single
     quotes, so they do not escape the insert statement."""
 
-    string = string.replace('"', "'")
-    return(string)
+    return string.replace('"', "'")
 
 
 def create_review_table(c):
-    """Creates a table called "Reviews" in the reviews.db databse if it doesn't
+    """Creates a table called "Reviews" in the reviews.db database if it doesn't
     already exist. If "Reviews" does exists, it informs the user that it will
     append the current database."""
 
-    try:
-        c.execute(f"""CREATE TABLE Reviews (
-                      book_auth TEXT(100) NOT NULL,
-                      book_title TEXT(100) NOT NULL,
-                      book_url TEXT(9999) NOT NULL,
-                      review_score INT(100),
-                      user_name TEXT(100) NOT NULL,
-                      review_date TEXT(100) NOT NULL
-                      );""")
-    except sqlite3.OperationalError as e:
-        print("Table already exists. The program will append the current table.")
+    c.execute("""CREATE TABLE IF NOT EXISTS Reviews (
+                 book_auth TEXT(100) NOT NULL,
+                 book_title TEXT(100) NOT NULL,
+                 book_url TEXT(9999) NOT NULL,
+                 review_score INT(100),
+                 user_name TEXT(100) NOT NULL,
+                 review_date TEXT(100) NOT NULL
+                 );""")
 
 
 def click_next_review_page(driver):
     """Finds a link on the page that loads the next set of thirty reviews. If
     it is unable to find the link it tells the user no link was found and allows
     the program to move on to the next book in the list."""
 
-    print("clicking link!")
+    # print("clicking link!")
```

Comment:
I just commented out most/all of the print statements so I could figure out what was going on